pkg/cluster/install.go

```diff
@@ -196,17 +194,17 @@ func (m *manager) bootstrap() []steps.Step {
 
 	if m.installViaHive {
 		s = append(s,
-			steps.Action(m.hiveCreateNamespace),
```

Comment:
The call was actually duplicated in the code block above this so it was effectively running twice for every installation. Removed it to avoid the extra overhead.